Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+// TODO: make this scroll!
+class PlaceDetailsCardView: UIView {
+
+    // TODO: accessibility labels (and parent view)
+    // TODO: set line height on all text. http://stackoverflow.com/a/5513730
+    lazy var titleLabel: UILabel = {
+        let view = UILabel()
+        view.textColor = Colors.detailsViewCardPrimaryText
+        view.font = Fonts.detailsViewTitleText
+        view.numberOfLines = 0
+        view.lineBreakMode = .byWordWrapping
+        return view
+    }()
+
+    lazy var categoryLabel: UILabel = {
+        let view = UILabel()
+        view.textColor = Colors.detailsViewCardPrimaryText
+        view.font = Fonts.detailsViewCategoryText
+        return view
+    }()
+
+    // TODO: Set styling.
+    lazy var urlLabel: UILabel = {
+        let view = UILabel()
+        view.textColor = .blue
```

Comment:
I know this is a default color, but I would still prefer to add it as a constant inside `Colors` so that it becomes super easy to change if necessary.